src/components/settings/settings-usage-consolidation/settings-usage-consolidation.js

```diff
@@ -66,6 +79,27 @@ const SettingsUsageConsolidation = ({
     label: month,
   }));
 
+  const parseUsageConsolidationId = value => {
+    if (customerKeyIsInvalid) {
+      clearUsageConsolidationErrors();
+    }
+
+    return value;
+  };
+
+  const defaultCurrency = {
+    value: '',
```

Comment:
Do we not need to set value to 'USD' here?

Reply:
That was timely value, because currency is required field